runtime/common/src/constants.rs

```diff
@@ -304,13 +304,21 @@ parameter_types! {
 
 // --- Stateful Storage Pallet ---
 parameter_types! {
-/// The maximum size of a page (in bytes) for an Itemized storage model
-pub const MaxItemizedPageSizeBytes: u32 = 4096;
-/// The maximum size of a page (in bytes) for a Paginated storage model
-pub const MaxPaginatedPageSizeBytes: u32 = 4096;
-/// The maximum size of a single item in an itemized storage model (in bytes)
-pub const MaxItemizedBlobSizeBytes: u32 = 512;
-/// The maximum number of pages in a Paginated storage model
-pub const MaxPaginatedPageCount: u32 = 64;
+	/// The maximum size of a page (in bytes) for an Itemized storage model
+	pub const MaxItemizedPageSizeBytes: u32 = 4096;
+	/// The maximum size of a page (in bytes) for a Paginated storage model
+	pub const MaxPaginatedPageSizeBytes: u32 = 4096;
+	/// The maximum size of a single item in an itemized storage model (in bytes)
+	pub const MaxItemizedBlobSizeBytes: u32 = 512;
+	/// The maximum number of pages in a Paginated storage model
+	pub const MaxPaginatedPageCount: u32 = 64;
+	/// The maximum number of actions in itemized actions
+	pub const MaxItemizedActionsCount: u32 = 7;
```

Comment:
I guess at some point we should make a more informed determination of what these limits should be?

Reply:
Correct! I think weights 2.0 and PoV size would have a direct effect on these numbers. So we have to make that decision after those are out.